apps/plugin-browser/src/pages/popup/popup-contents/action-center/log/inference-request.tsx

```diff
@@ -34,59 +35,76 @@ const MetadataItem = ({ label, value }: { label: string; value: string }) => (
   </Stack>
 );
 
+const generateDurationString = (interval: Interval) =>
+  formatDuration(intervalToDuration(interval));
+
 const InferenceMetadata = ({ request }: { request: PageEntityInference }) => {
+  const [timeElapsed, setTimeElapsed] = useState(() =>
+    generateDurationString({
+      start: new Date(request.createdAt),
+      end: new Date(request.finishedAt || Date.now()),
+    }),
+  );
+
   const usage =
     "data" in request &&
     request.data.contents[0]?.usage.reduce(
       (acc, usageItem) => acc + usageItem.total_tokens,
       0,
     );
 
-  const duration = request.finishedAt
-    ? formatDuration(
-        intervalToDuration({
-          start: new Date(request.createdAt),
-          end: new Date(request.finishedAt),
-        }),
-      )
-    : null;
+  useEffect(() => {
+    if (!request.finishedAt) {
+      setTimeout(() => {
+        setTimeElapsed(
+          generateDurationString({
+            start: new Date(request.createdAt),
+            end: new Date(request.finishedAt || Date.now()),
+          }),
+        );
+      }, 1_000);
+    }
+  });
 
   return (
-    <Stack
-      alignItems="center"
-      direction="row"
-      justifyContent="space-between"
-      sx={() => ({
-        borderTopWidth: 1,
-        borderTopStyle: "solid",
-        ...borderColors,
-        pt: 0.5,
-      })}
-    >
-      <Box
-        component="a"
-        href={request.sourceUrl}
-        sx={({ palette, transitions }) => ({
-          color: palette.blue[70],
-          textDecoration: "none",
-          fontSize: metadataFontSize,
-          fontWeight: 600,
-          lineHeight: 1.5,
-          marginBottom: "1px",
-          opacity: 0.7,
-          transition: transitions.create("opacity"),
-          "&:hover": {
-            opacity: 1,
-          },
+    <Box>
+      <Stack
+        alignItems="center"
+        direction="row"
+        justifyContent="space-between"
+        sx={() => ({
+          borderTopWidth: 1,
+          borderTopStyle: "solid",
+          ...borderColors,
+          pt: 0.5,
         })}
-        target="blank"
       >
-        View source page
-      </Box>
-      <MetadataItem label="Model" value={request.model} />
-      {usage && <MetadataItem label="Tokens" value={usage.toString()} />}
-      {duration && <MetadataItem label="Time" value={duration} />}
-    </Stack>
+        <Box
+          component="a"
+          href={request.sourceUrl}
+          sx={({ palette, transitions }) => ({
+            color: palette.blue[70],
+            textDecoration: "none",
+            fontSize: metadataFontSize,
+            fontWeight: 600,
+            lineHeight: 1.5,
+            marginBottom: "1px",
+            opacity: 0.7,
+            transition: transitions.create("opacity"),
+            "&:hover": {
+              opacity: 1,
+            },
+          })}
+          target="blank"
+        >
```

Comment:
I've updated this in https://github.com/hashintel/hash/pull/3905/commits/f222b00d2bace8e60433c008640ea45bbe5a927d

It gains an underline but that's fine since we use it elsewhere